tests/plugins/list/blockconversion.js

```diff
@@ -0,0 +1,57 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: list,justify,bidi,table,forms,toolbar */
+
+bender.editors = {
+	classic: {
+		creator: 'replace',
+		name: 'classic'
+	},
+	divarea: {
+		name: 'divarea',
+		creator: 'replace',
+		config: {
+			extraPlugins: 'divarea'
+		}
+	},
+	inline: {
+		name: 'inline',
+		creator: 'inline'
+	}
+};
+
+var tests = {
+	'test conversion of embedded unordered list and paragraphs to numbered list': function( editor, bot ) { // Strange test result for: 0, 8
```

Comment:
`Strange test result for: 0, 8` comment will look much better in a line below. Here it exceeds 120 chars which, if convenient, we like to avoid.